ros-build-essential/ros-build-essential.spec

```diff
@@ -0,0 +1,27 @@
+Name: ros-build-essential
+Version: 1.0.0
+Summary: Packages expected when building any ROS package
+Release: 1%{?dist}%{?release_suffix}
+BuildArch: noarch
+License: ASL 2.0
+URL: https://github.com/ros-infrastructure/infra-variants
+Source0: README.md
+Source1: copyright
+
+# TODO add packages from mock's default configuration.
+Requires: cmake, g++, git, make, python3
```

Comment:
Should we resolve this TODO?  Also, the README mentions the `Development Tools` group, but I don't see that as a dependency here.  Should we add it?

Reply:
Groups are more of a yum/dnf concept than an RPM concept and you can't take a dependency on a group at this level.

Here is the list of buildroot packages for RHEL (AlmaLinux): https://github.com/rpm-software-management/mock/blob/0e7e8db8489563cbd6a21378860f45828c25a99b/mock-core-configs/etc/mock/templates/almalinux-8.tpl#L1

For Fedora, it's defined in the repository metadata "comps" file. For Fedora 36, you'll find it here: https://dl.fedoraproject.org/pub/fedora/linux/releases/36/Everything/x86_64/os/repodata/3f69beebaa5fb330617ca37b79c7e6381a415957f2999bc141386a1271ec86bc-comps-Everything.x86_64.xml.xz

Here's the uncompressed XML snippet that defines the list:
<details>

```xml
<packagelist>
<packagereq type="mandatory">bash</packagereq>
<packagereq type="mandatory">bzip2</packagereq>
<packagereq type="mandatory">coreutils</packagereq>
<packagereq type="mandatory">cpio</packagereq>
<packagereq type="mandatory">diffutils</packagereq>
<packagereq type="mandatory">fedora-release-common</packagereq>
<packagereq type="mandatory">findutils</packagereq>
<packagereq type="mandatory">gawk</packagereq>
<packagereq type="mandatory">glibc-minimal-langpack</packagereq>
<packagereq type="mandatory">grep</packagereq>
<packagereq type="mandatory">gzip</packagereq>
<packagereq type="mandatory">info</packagereq>
<packagereq type="mandatory">patch</packagereq>
<packagereq type="mandatory">redhat-rpm-config</packagereq>
<packagereq type="mandatory">rpm-build</packagereq>
<packagereq type="mandatory">sed</packagereq>
<packagereq type="mandatory">shadow-utils</packagereq>
<packagereq type="mandatory">tar</packagereq>
<packagereq type="mandatory">unzip</packagereq>
<packagereq type="mandatory">util-linux</packagereq>
<packagereq type="mandatory">which</packagereq>
<packagereq type="mandatory">xz</packagereq>
</packagelist>
```

</details>

Reply:
Done in [ef2cc37](https://github.com/ros-infrastructure/infra-variants/pull/1/commits/ef2cc3717d33cac5c4fcd0f44029a2fa02b01de0)